### PR TITLE
Add image build config for ols console plugin

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@kyoto fyi, trying to add the image build config here.  you may know better than I what is needed though.

**Comment by bparees** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by bparees** [comment]
@kyoto ok this is building successfully, for whatever that is worth :)

take a look and if it makes sense to you let's merge it, if not let's discuss.  I took some guesses on the base/builder images.

File: ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    namespace: openshift
+    tag: latest
+images:
+- dockerfile_path: Dockerfile
+  from: ubi-9-nginx-120
+  inputs:
+    nodejs-16:
+      as:
+      - build
+  to: ols-console
```
Comment (kyoto) [inline ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml:22]: Is `ols-console` correct? I didn't see this name used elsewhere.

**Comment by kyoto** [comment]
Thanks @bparees. I'm not familiar with all the options, but I didn't see anything that looked wrong apart from one small question. Generally LGTM.

File: ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml (hunk 1 above)
Comment (bparees) [reply ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml:22]: hm, good catch, i think lightspeed-console-plugin is what this should be, i'll fix it.

**Comment by bparees** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Pull request merged**